pkg/ddc/alluxio/ufs.go

```diff
@@ -18,12 +18,20 @@ package alluxio
 
 import (
 	"fmt"
-
 	datav1alpha1 "github.com/fluid-cloudnative/fluid/api/v1alpha1"
 	"github.com/fluid-cloudnative/fluid/pkg/utils"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"os"
 )
 
+func IsMountWithConfigMap() bool {
+	if envVal, exists := os.LookupEnv(MountConfigStorage); exists {
```

Comment:
For runtime controller, the env MountConfigStorage is always configmap, and it does not change with different runtimes. So it is need more information to judge if the runtime IsMountWithConfigMap.